src/handling/world/CharacterTransfer.java

```diff
@@ -32,7 +32,7 @@
 
     public int characterid, accountid, fame, pvpExp, pvpPoints, hair, face,
             faceMarking, elf, mapid, honourexp, honourlevel, guildid,
-            partyid, messengerid, ACash, nxCredit, MaplePoints,
+            partyid, messengerid, ACash, nxCredit, MaplePoints, RewardPoints,
```

Comment:
This has been since been corrected, thanks! I thought so, too, but again stuck with it for consistency.